packages/graphiql-react/vite.config.js

```diff
@@ -1,42 +0,0 @@
-import { defineConfig } from 'vite';
-import react from '@vitejs/plugin-react';
-import reactSvgPlugin from 'vite-plugin-react-svg';
-import postCssNestingPlugin from 'postcss-nesting';
-// import noBundlePlugin from 'vite-plugin-no-bundle';
-export default defineConfig({
-  plugins: [
-    // TODO:  disable bundling dependencies
-    // noBundlePlugin(),
```

Comment:
Not sure why we'd need this, we already exclude dependencies that shouldn't be bundled with `build.rollupOptions.external` 🤔